fedmsg.d/fedimg.py

```diff
@@ -27,7 +27,7 @@
 NUM_PORTS = 2 * ((NUM_BASE_THREADS + NUM_ATOMIC_THREADS) + 1)
 
 config = dict(
-    kojiconsumer=True,
+    fedimgconsumer=True,
```

Comment:
Will need to remember to change this in our ansible configuration too when this gets deployed.